src/applications/caregivers/actions/fetchFacilities.js

```diff
@@ -83,12 +90,18 @@ export const fetchFacilities = async ({
 
         // Return a new facility object with the updated address
         return {
-          ...facility,
+          ...attributes,
+          id: facility.id,
```

Comment:
The id is not passed in the response object `attributes`, so I am adding it to the response here.